gaphor/ui/csscompletion.py

```diff
@@ -5,6 +5,233 @@
 from gaphor.core.styling import Style
 
 
+class CssNamedColorsCompletionProvider(GObject.GObject, GtkSource.CompletionProvider):
+    # https://www.w3.org/TR/css-color-4/
+    COLORS = [
+        "aliceblue",
```

Comment:
We can take the colors from `tinycss2.color3._COLOR_KEYWORDS` as well. That's where the named colors are defined that can be translated. I suppose it's the same set as you defined here.